CHANGELOG.md

```diff
@@ -68,6 +69,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Added a `language` field to the `Account`. Consequently, the `account` and
   `accountList` API responses now include this field. The `insertAccount` and
   `updateAccount` GraphQL API endpoints are also updated to support the field.
+- Added logic to check the `lastSigninTime` value in the `signIn` GraphQL API.
```

Comment:
I think we need to document how the new logic works, as well as the fact that it has changed, because in this case, the new behavior is the primary concern for users.

Reply:
Added content.

Reply:
The key point is that this API rejects a user's access if it is their first attempt. Therefore, it's not enough to simply mention checking the `lastSigninTime`. Additionally, the word "logic" seems unnecessary. I suggest:

- Changed the `signIn` to reject the first sign-in attempt unless it is done through the new `signInWithNewPassword` API.

In this case, "Changed" or "Updated" is preferred over "Added" because the behavior of the API has actually changed from the user's perspective. Also, the word "reject" or any similar one should be included to convey the key point.

Reply:
I will delete the explanation below and change it to the sentence you suggested.
**"Apply a requirement for a password change upon the initial `signIn`."**